cluster-autoscaler/cloudprovider/ionoscloud/README.md

```diff
@@ -32,3 +38,12 @@ Edit [`cluster-autoscaler-standard.yaml`](./examples/cluster-autoscaler-standard
 ```console
 kubectl apply -f examples/cluster-autoscaler-standard.yaml
 ```
+
+## Development
+
+The unit tests use mocks generated by [mockery](https://github.com/vektra/mockery) v2. To update them run:
+
+```sh
+mockery --inpackage --testonly --case snake --boilerplate-file ../../../hack/boilerplate/boilerplate.generatego.txt --name APIClient
+mockery --inpackage --testonly --case snake --boilerplate-file ../../../hack/boilerplate/boilerplate.generatego.txt --name IonosCloudManager
```

Comment:
Now you can refer to `make generate-mocks` ;)

Reply:
damn you `*`